app/models/reservation.rb

```diff
@@ -2,16 +2,14 @@
 
 class Reservation < ApplicationRecord # :nodoc:
   # Enums
-  enum status: %i[pending confirmed canceled]
+  enum reservation_status: %i[pending confirmed canceled]
```

Comment:
Why switching the name? if `status` it is an attribute within `Reservation` I would say the prefix is redundant.

I think in here, there might also be some rethink needed. How do the statuses work? Can you cancel a confirmed reservation? can you confirm a canceled reservation?

Reply:
I agree, there is no need to prefix "reservation" to status.

In terms of state changes (transitions), we could in the future think about `state_machine` if we want to restrict state transitions. Also good to learn about using state_machines

Reply:
I got an error when tried to create  a reservation or guest, due to they have the same attributes, I'll redo the test without the prefix. Thanks!